test/Microsoft.Azure.SignalR.Tests/Infrastructure/TestClientConnectionManager.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.SignalR.Tests.Infrastructure
+{
+    class TestClientConnectionManager : IClientConnectionManager
```

Comment:
We can just use `ClientConnectionManager` right? So this class is not necessary.